addons/script.module.pil/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.pil"
        name="Python Image Library"
-       version="3.0.0"
+       version="7.1.2"
```

Comment:
At least 7.1.2 is guaranteed for all platforms?

Reply:
How best to handle this? Nothing's really guaranteed on non depends platforms,.ie anything linux. Ubuntu 18 ships the 5.1.0, do I reduce to that?
Does it really matter what this sits at? Like does it have an affect, or is it just cosmetic in add-on info?

Reply:
Just dropped back to 5.1.0. Lowest common denominator of supported platforms i guess

Windows - 6.2.1
Apple - 7.1.2
Android - 7.1.2
Linux - whatever they ship i guess. Ubuntu 18.04 - 5.1.0